pkg/services/navtree/navtreeimpl/applinks.go

```diff
@@ -267,7 +267,7 @@ func (s *ServiceImpl) readNavigationSettings() {
 		"grafana-ml-app":                   {SectionID: navtree.NavIDAlertsAndIncidents, SortWeight: 3, Text: "Machine Learning"},
 		"grafana-cloud-link-app":           {SectionID: navtree.NavIDCfg},
 		"grafana-easystart-app":            {SectionID: navtree.NavIDRoot, SortWeight: navtree.WeightApps + 1, Text: "Connections", Icon: "adjust-circle"},
-		"grafana-k6-app":                   {SectionID: navtree.NavIDRoot, SortWeight: navtree.WeightAlertsAndIncidents + 1, Text: "Performance testing", Icon: "k6"},
+		"k6-app":                           {SectionID: navtree.NavIDRoot, SortWeight: navtree.WeightAlertsAndIncidents + 1, Text: "Performance testing", Icon: "k6"},
```

Comment:
I feel like these should both be in here until no one is using `grafana-k6-app` in Grafana Cloud? @ashharrison90 can confirm, but I don't think there's downsides to having extras in here - plugins not installed are just ignorred.

Reply:
discussing in slack rn: https://raintank-corp.slack.com/archives/C01C4K8DETW/p1675160311952259

i'm not so confident about having both in here at the same time - not sure exactly what order the plugins get processed in, and since both _could_ be available there might be some unintended consequences.

think the safest thing to do is create a new feature toggle and use it to gate this change? 🤔

Reply:
I think it'd be better if the old plugin with id `grafana-k6-app` ends up under apps instead of having two k6 plugins under the top-level Performance testing section.